pkg/manager/config/config.go

```diff
@@ -403,6 +406,11 @@ func (i *Instance) GetParallelTasksWithAutoDetection() int {
 	return parallelTasks
 }
 
+func (i *Instance) GetPreviewAudio() bool {
+	viper.SetDefault(PreviewAudio, previewAudioDefault)
```

Comment:
This introduces a race condition that can crash the thread and server during preview generation. Crash log is below.

```
fatal error: concurrent map writes
fatal error: concurrent map writes

goroutine 341402 [running]:
runtime.throw(0x15c7aa9, 0x15)
        /usr/local/go/src/runtime/panic.go:774 +0x72 fp=0xc016853100 sp=0xc0168530d0 pc=0x436b32
runtime.mapassign_faststr(0x13d1e60, 0xc000374990, 0x15b0928, 0xd, 0xc000374990)
        /usr/local/go/src/runtime/map_faststr.go:291 +0x3fe fp=0xc016853168 sp=0xc016853100 pc=0x41ac4e
github.com/spf13/viper.(*Viper).SetDefault(0xc000289680, 0x15b0928, 0xd, 0x137e0e0, 0x3063b01)
        /stash/vendor/github.com/spf13/viper/viper.go:1304 +0x1bf fp=0xc0168531f8 sp=0xc016853168 pc=0xce154f
github.com/spf13/viper.SetDefault(...)
        /stash/vendor/github.com/spf13/viper/viper.go:1293
github.com/stashapp/stash/pkg/manager/config.(*Instance).GetPreviewAudio(0xc0004ae1e0, 0x7)
        /stash/pkg/manager/config/config.go:410 +0x68 fp=0xc016853230 sp=0xc0168531f8 pc=0xceb268
github.com/stashapp/stash/pkg/manager.(*GeneratePreviewTask).Start(0xc00f22e240, 0xc020ed1120)
        /stash/pkg/manager/task_generate_preview.go:54 +0x47d fp=0xc016853f38 sp=0xc016853230 pc=0x1158f4d
github.com/stashapp/stash/pkg/manager.(*singleton).Generate.func1.4()
        /stash/pkg/manager/manager_tasks.go:291 +0x33 fp=0xc016853f58 sp=0xc016853f38 pc=0x116bd33
github.com/stashapp/stash/pkg/job.(*Progress).ExecuteTask(0xc0003cc100, 0xc00f99c7e0, 0x8f, 0xc01a3e3580)
        /stash/pkg/job/progress.go:137 +0xb3 fp=0xc016853fc0 sp=0xc016853f58 pc=0xd8fdb3
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1357 +0x1 fp=0xc016853fc8 sp=0xc016853fc0 pc=0x4668f1
created by github.com/stashapp/stash/pkg/manager.(*singleton).Generate.func1
        /stash/pkg/manager/manager_tasks.go:290 +0xf07
```